doc/install_doc_component.html

```diff
@@ -0,0 +1,170 @@
+      Terminal)</strong></p>
+      <p>Then type the following line and press "Enter"</p>
+      <div class="code"><pre>pip install -U --user nilearn</pre></div>
+
+      <p class="emphasize">Third: open IPython</p>
+      <p><strong>(You can open it by writing "ipython" in the terminal and
+      pressing "Enter")</strong></p>
+      <p>Then type in the following line and press "Enter":</p>
+      <div class="code"><pre>In [1]: import nilearn</pre></div>
+      <p>If no error occurs, you have installed nilearn correctly.</p>
+
+    </li>
+
+    <li>
+
+      <div class="contents"><p>If you are <strong>Ubuntu or Debian</strong>
+      and you have access to <a class="reference external"
+      href="http://neuro.debian.net/pkgs/python-nilearn.html" target="_blank">
+      Neurodebian</a> or can install it. Then simply install nilearn through
+      <a class="reference external"
+      href="http://neuro.debian.net/pkgs/python-nilearn.html" target="_blank">Neurodebian</a>.</p></div>
+
+      <p class="emphasize">First: Install dependencies</p>
+
+      <p>Install or ask your system administrator to install the following
+      packages using the distribution package manager: <strong>ipython</strong>
+      , <strong>scikit-learn</strong> (sometimes called <cite>sklearn</cite>,
+      or <cite>python-sklearn</cite>), <strong>matplotlib</strong> (sometimes
+      called <cite>python-matplotlib</cite>) and <strong>nibabel</strong> (
+      sometimes called <cite>python-nibabel</cite>)</p>
+
+      <p><strong>If you do not have access to the package manager we recommend
+      that you install a complete scientific Python distribution like 64 bit
+      <a class="reference external"
+      href="https://store.continuum.io/cshop/anaconda/" target="_blank">
+      Anaconda</a></strong>. Since It will meet all the requirements of
+      nilearn, save you time and trouble.</p>
+
+      <p class="emphasize">Second: open a Terminal</p>
+      <p><strong>(Press ctrl+alt+t and a Terminal console will pop up)</strong></p>
+      <p>Then type the following line and press "Enter"</p>
+      <div class="code"><pre>pip install -U --user nilearn</pre></div>
+
+      <p class="emphasize">Third: open IPython</p>
+      <p><strong>(You can open it by writing "ipython" in the terminal and
+      pressing "Enter")</strong></p>
+      <p>Then type in the following line and press "Enter":</p>
+      <div class="code"><pre>In [1]: import nilearn</pre></div>
+      <p>If no error occurs, you have installed nilearn correctly.</p>
+
+    </li>
+
+    <li>
+
+      <p class="emphasize">To Install the development version:</p>
+
+      <p><strong>Use git as an alternative to using pip, to get the latest
+      nilearn version</strong></p>
+      <p class="first">Simply run the following command (as a shell command,
+      not a Python command):</p>
+      <div class="code"><pre>git clone https://github.com/nilearn/nilearn.git
+      </pre></div>
```

Comment:
You probably don't want to a new line inside the <pre> because the new line is visible in the text box:

![additional_new_line_in_text_box](https://cloud.githubusercontent.com/assets/1680079/8729606/65559182-2bec-11e5-92b6-1a9f155d4693.png)